src/Utility/TensorFunction.C

```diff
@@ -23,7 +23,17 @@ utl::matrix3d<Real> TensorFunc::gradient(const Vec3& X) const
 }
 
 
-Tensor TensorFunc::timeDerivative(const Vec3& X) const
+utl::matrix4d<Real> TensorFunc::hessian (const Vec3& X) const
+{
+  const size_t nsd = sqrt(ncmp);
+  utl::matrix4d<Real> result(nsd, nsd, nsd, nsd);
+  result.fill(this->evalHessian(X).data());
+
```

Comment:
here also the blank line should go, for the consistency

Reply:
ok